packages/theme-dark-extension/style/urls.css

```diff
@@ -62,7 +62,8 @@
   --jp-icon-inverse-circle: url('icons/md/circle-inverse.svg');
   --jp-icon-close-circle: url('icons/md/close-circle.svg');
   --jp-icon-inverse-close-circle: url('icons/md/close-circle-inverse.svg');
-  --jp-icon-search: url('icons/md/search.svg');
+  --jp-icon-search: url('icons/md/search-white.svg');
+  --jp-icon-search-white: url('icons/md/search.svg');
```

Comment:
This is confusing to me. Why is `--jp-icon-search` pointing to `search-white.svg` and why is `--jp-icon-search-white` pointing to `search.svg`?

Reply:
Perhaps it should be named `--jp-icon-inverse-search`, etc?

Reply:
@ian-r-rose They're swapped in dark theme for when the background color inverts. I can definitely change it to --jp-icon-inverse-search if you think that would be more intuitive!